netbox_librenms_plugin/views/base/interfaces_view.py

```
@@ -1,11 +1,16 @@
+import logging
+
 from django.contrib import messages
 from django.core.cache import cache
-from django.shortcuts import get_object_or_404, redirect, render
+from django.shortcuts import get_object_or_404, redirect
```

Flake8 (7.3.0)

[error] 'django.shortcuts.redirect' imported but unused

(F401)